tests/lora/test_lora_layers_af.py

```diff
@@ -0,0 +1,114 @@
+# coding=utf-8
```

Comment:
Let's rename this file to `test_lora_layers_auraflow.py`.

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/6514892fd53f06502329b3927714f7d166db89fd